odoo/addons/base/models/ir_http.py

```diff
@@ -250,9 +253,6 @@ def _generate_routing_rules(cls, modules, converters):
 
     @classmethod
     def routing_map(cls, key=None):
-        if not hasattr(cls, '_routing_map'):
-            cls._routing_map = {}
-            cls._rewrite_len = {}
```

Comment:
Hello @JKE-be may I ask you to check if this fix makes sense ?

Reply:
Probably not wrong ... but why do you need to change it ? 
Maybe it hide another bug?

Reply:
> Probably not wrong ... but why do you need to change it ?
> Maybe it hide another bug?
Yep, when a the test class TestQwebProcessAtt is run alone, it crashes (trying to access _rewrite_len attribute). The bug is hidden on runbot because another test calls routing_map before and sets the attributes.